nixos/modules/services/networking/syncthing.nix

```diff
@@ -34,11 +34,9 @@ in
       };
 
       dataDir = mkOption {
-        default = "/var/lib/syncthing";
+        default = "~/.config/syncthing";
```

Comment:
i think the default is still better to have ``/var/lib/syncthing``, no? otherwise you will end up storing into /root/.config/syncthing